metrics/grafana/pd.json

```diff
@@ -4712,7 +4699,7 @@
           "timeFrom": null,
           "timeRegions": [],
           "timeShift": null,
-          "title": "Store write rate keys",
+          "title": "Store write qps",
```

Comment:
qps may be not a good name since the unit is ops.

Reply:
how about `Store write query`?

Reply:
how about `Store write query`?

Reply:
better than qps